goveralls.go

```diff
@@ -82,6 +84,7 @@ type SourceFile struct {
 type Job struct {
 	RepoToken          *string       `json:"repo_token,omitempty"`
 	ServiceJobId       string        `json:"service_job_id"`
+	ServiceJobNumber   string        `json:"service_job_number,omitempty"`
```

Comment:
Its name is `service_number` on the document https://docs.coveralls.io/api-reference, but doesn't work.
It seems to be a typo.

https://github.com/nickmerwin/node-coveralls/blob/2ed185ca3626591549ed33c2363a47b824a39d0f/lib/getOptions.js#L80